database/factories/ModelFactory.php

```diff
@@ -56,6 +70,19 @@
 
 $factory->define(App\Model\Favorite::class, function (Faker $faker) {
     return [
+        'favoritable_id'   => rand(1,15),
+        'favoritable_type' => $faker->randomElement(['books', 'comments', 'posts'])
```

Comment:
- please check values of favoritable_type? normally, values is a namespace of class.